### PR TITLE
chore: MapControl Sample - Update to net6

### LINKED ISSUE #296: Replace the MapControl to Mapsui on MapControl Sample
After the update to .NET 6.0 the MapControl becomes incompatible with the sample. The suggestion comes from this comment
https://github.com/unoplatform/Uno.Samples/pull/279#issuecomment-1375905288

### PR DESCRIPTION
**Pull request by aesteves900**
After the update for .NET6 the legacy MapControl becomes incompatible, then the solution was  replace from MapControl to Mapsui compatible with WinUI and  .NET 6  
Closes #296

### CODE REVIEW

**Comment by nickrandolph** [comment]
@jeromelaban should we convert this sample to use something like mapsui? or should we just archive this sample

**Comment by jeromelaban** [comment]
Let's convert it to use mapsui, now that it's supported!

File: UI/MapControlSample/MapControlSample/MapControlSample.Mobile/Android/Assets/AboutAssets.txt [hunk 1]
```diff
@@ -0,0 +1,22 @@
+To add cross-platform image assets for your Uno Platform app, use the Assets folder
+in the shared project instead. Assets in this folder are Android-only assets.
+
+Any raw assets you want to be deployed with your application can be placed in
+this directory (and child directories) and given a Build Action of "AndroidAsset".
+
+These files will be deployed with you package and will be accessible using Android's
```
Comment (agneszitte-nventive) [inline UI/MapControlSample/MapControlSample/MapControlSample.Mobile/Android/Assets/AboutAssets.txt:7]:
```suggestion
These files will be deployed with your package and will be accessible using Android's
```

File: UI/MapControlSample/MapControlSample/MapControlSample.Shared/App.xaml.cs [hunk 2]
```diff
@@ -126,6 +116,14 @@ private void OnSuspending(object sender, SuspendingEventArgs e)
         /// </summary>
         private static void InitializeLogging()
         {
+#if DEBUG
+            // Logging is disabled by default for release builds, as it incurs a significant
+            // initialization cost from Microsoft.Extensions.Logging setup. If startup performance
+            // is a concern for your application, keep this disabled. If you're running on web or 
+            // desktop targets, you can use url or command line parameters to enable it.
+            //
+            // For more performance documentation: https://platform.uno/docs/articles/Uno-UI-Performance.html
```
Comment (agneszitte-nventive) [inline UI/MapControlSample/MapControlSample/MapControlSample.Shared/App.xaml.cs:125]:
```suggestion
            // Logging is disabled by default for release builds, as it incurs a significant
            // initialization cost from Microsoft.Extensions.Logging setup. If startup performance
            // is a concern for your application, keep this disabled. If you're running on the web or 
            // desktop targets, you can use URL or command line parameters to enable it.
            //
            // For more performance documentation: https://platform.uno/docs/articles/Uno-UI-Performance.html
```

File: UI/MapControlSample/MapControlSample/MapControlSample.Shared/App.xaml.cs [hunk 3]
```diff
@@ -117,7 +107,7 @@ void OnNavigationFailed(object sender, NavigationFailedEventArgs e)
         private void OnSuspending(object sender, SuspendingEventArgs e)
         {
             var deferral = e.SuspendingOperation.GetDeferral();
-            // TODO: Save application state and stop any background activity
+            //TODO: Save application state and stop any background activity
```
Comment (agneszitte-nventive) [inline UI/MapControlSample/MapControlSample/MapControlSample.Shared/App.xaml.cs:110]:
```suggestion
            //TODO: Save the application state and stop any background activity
```

**Comment by nickrandolph** [comment]
@aesteves900 did you get this to work for ios and android? I've added gtk and wpf target platforms and they work but I can't get android or ios to work

**Comment by nickrandolph** [comment]
Pushed this back to draft until issues with mapsui for ios/android are resolved

**Comment by aesteves900** [comment]
> @aesteves900 did you get this to work for ios and android? I've added gtk and wpf target platforms and they work but I can't get android or ios to work

@nickrandolph , I tested on Android and works properly, unfortunately, I have no device to test on iOS. I do double -check on 
Android.
OS

**Review by nickrandolph — APPROVED** [review]

**Pull request merged**